frontend/pages/ManageControlsPage/OSUpdates/components/AppleOSTargetForm/AppleOSTargetForm.tsx

```
@@ -214,8 +219,8 @@ const AppleOSTargetForm = ({
         disabled={gitOpsModeEnabled}
         name="deadline"
         label="Deadline"
-        tooltip="The end user can't dismiss the OS update once they reach this deadline. Deadline is 12:00 (Noon), the host's local time."
-        helpText="YYYY-MM-DD format only (e.g., “2024-07-01”)."
+        tooltip="The end user can&apos;t dismiss the OS update once they reach this deadline. If no time is specified, defaults to 12:00 (noon) local time."
```

ESLint

[error] Replace `&apos;` with `'`

(prettier/prettier)